dev/com.ibm.ws.ssl/src/com/ibm/ws/ssl/config/ProtocolHelper.java

```diff
@@ -67,18 +68,38 @@ public void checkProtocolValueGood(String sslProtocol) throws SSLException, Unsu
         String[] protocols = sslProtocol.split(",");
 
         if (protocols.length > 1) {
```

Comment:
Does some logic need to be added to the else of this if where protocols length is 1 so that we put out an error if the single protocol provided by the user is NOT TLSv1.2 or TLSv1.3?

Reply:
I don't know but will look into it later on.

Reply:
I will look into it later
https://github.com/OpenLiberty/open-liberty/issues/30583